components/organisms/Problem.tsx

```diff
@@ -22,10 +22,12 @@ export default function Problem(props: Props) {
       className={classes.container}
       maxWidth="md"
     >
-      <Typography variant="h4" gutterBottom={true}>
-        {title}
+      <Typography variant="body1" component="section">
```

Comment:
コンテンツとして独立しているのであればarticleのほうがより適切でしょうか？
```suggestion
      <Typography variant="body1" component="article">
```

Reply:
titleやchildrenを受け取る設計上、このコンポーネント自身がコンテンツとして独立しているか否か決定することは困難ではないでしょうか。

Reply:
たしかに、このコンポーネント単体ではどのようなコンテンツが流し込まれるか分かりませんね、おっしゃるとおりです